andela/storm-backend
# setup morgan for logging

#### What this PR does
This PR will setup morgan for logging

#### Tasks completed
- install morgan-body

#### How this can be manually tested
- Clone this branch
- Change directory into it
- Run `npm install` in your terminal to install all the dependencies
- Add all the environment variables as specified in the `.env.config` file
- Run `npm run start:dev` in your terminal to start the development server
- Test with postman or go to this URL to test with swagger `http://localhost:3000/api/v1/docs`
- Hit an endpoint and check your terminal to see the log output

#### What are the relevant pivotal tracker stories?
[ #168402426](https://www.pivotaltracker.com/story/show/168402426)
